backend_site/settings.py

```diff
@@ -0,0 +1,44 @@
+"""Configuration settings for this project"""
+
+from os import path, pardir
+
+BACKEND_DIR = path.abspath(path.dirname(__file__))  # This directory
+PROJECT_ROOT = path.abspath(path.join(BACKEND_DIR, pardir))
+
+STATIC_DIR = path.join(PROJECT_ROOT, 'frontend-site', 'dist')
+STATIC_ROUTE = '/static/'
+
+
+class Config:
+    """Base configuration."""
+    BACKEND_DIR = BACKEND_DIR
+    PROJECT_ROOT = PROJECT_ROOT
+    # DEBUG_TB_INTERCEPT_REDIRECTS = False
+    CACHE_TYPE = 'simple'  # Can be "memcached", "redis", etc.
+    CORS_ORIGIN_WHITELIST = [
+        'http://0.0.0.0:5000',
```

Comment:
Could the config be changed based on the environment mode? For example, `dev` mode could have all the `localhost` and `0.0.0.0` listings, while the `prod` mode would have all of the sites that are whitelisted in the production version (like `lug.cs.uic.edu`).

Additionally, if we're going to be adding `localhost` and `0.0.0.0` for development purposes, can you add port 8080 as well? The default port for development on the frontend is 8080 (assuming nothing is taking up that port), and I get the error below on the current config when running `npm run serve` for the frontend site for development.


![image](https://user-images.githubusercontent.com/16447522/51509868-5ba55900-1dc0-11e9-926c-25c49de2fdbd.png)

Reply:
The error says "No 'Access-Control-Allow-Origin' header is present", so there's no CORS header at all on the response? Which could be a problem either with the request going to the wrong place or the library not being set up right. I'll look into it in the morning.